prlworkflows/tests/test_sqs.py

```diff
@@ -235,7 +235,9 @@ def test_abstract_sqs_scales_volume_when_made_concrete():
 
     structure = lat_in_to_sqs(ATAT_FCC_L12_LATTICE_IN)
     concrete_structure = structure.get_concrete_sqs([['Fe', 'Ni'], ['Al']])
-    assert np.isclose(concrete_structure.volume, 421.08774505083824)
+    concrete_structure = structure.get_concrete_sqs([['Fe', 'Ni'], ['Al']])
+    assert np.isclose(concrete_structure.volume, 445.35213050176463)
+    assert np.isclose(concrete_structure.density, 4.122750000000001)
```

Comment:
In the same vein as above, are the density and volume here the same as you calculated by hand for this structure? Given the sorting issue, you should check by hand and make sure you are calculating what you think you are.